metadata-checker/README.md

```diff
@@ -0,0 +1,9 @@
+# metadata-checker
+
+## What it does
+
+This example adds a popup that lets check the opengraph, twitter and facebook metadata from the current web page.
```

Comment:
Please use correct syntax and capitalization for things like Open Graph, Twitter, Facebook (and it would be helpful to link to the docs for their metadata too).